pages/code_schools.js

```diff
@@ -203,6 +212,22 @@ export default class CodeSchools extends React.Component {
             </FlatCard>
           ))}
         />
+
+        <Modal
+          isOpen={isModalOpen}
+          screenReaderLabel="Campus locations"
+          onRequestClose={this.handleModalToggle}
+          className={styles.schoolLocationModal}
+        >
+          <>
+            {isModalOpen && <h3>{state.modalOpen.name} Campuses</h3>}
```

Comment:
Do we need to short-circuit anything?

Reply:
Not sure what you mean by short-circuit here.

Reply:
The act of `boolean && JSX` is what I colloquially refer to as "short-circuiting".

My mindset is that the modal itself is either there or not, so why worry about its content? Maybe define initial state more to ensure you don't have `Cannot read property 'name' of undefined`